content/about/index.md

```diff
@@ -11,21 +11,23 @@ weight = 10
 
 ## About SVNA
 
-The Santiago Villa Residents' Association (SVRA) was started by Barbara Davis, Trey Bornmann, and Bee Hanson. The current organization was started in 2015, but there have been other informal residents organizations. We still have some typewritten directories from those early years.
+The Santiago Villa Residents' Association (SVRA) was started by Alex Brown, Bee Hanson, Barbara Davis, Stuart Esbit, and Trey Bornmann. That organization was started in 2015, but over the years there have been other informal resident organizations. We still have some typewritten directories from those early years.
 
-SVRA was started by by compiling a directory of neighbors, and in 2017, our membership was big enough to apply for and receive a grant from the City of Mountain View. At that time, our Residents' Association became the Santiago Villa Neighborhood Association (SVNA).
+SVRA was started by gradually making a directory of neighbors. By 2017, our membership was big enough to apply for and receive a grant from the City of Mountain View. That's when our Residents' Association became the Santiago Villa Neighborhood Association (SVNA). Our SVNA Google Group has 222 records, and it grows each time we have a new event.
```

Comment:
I wouldn't put the granular google group data in.

Reply:
OK, I took out the SVNA Google Group sentence.